### PR TITLE
Add 'woocommerce_review_order_before_payment_methods' action in 'payments.php'

### PR DESCRIPTION
**Pull request by manospsyx**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Added `woocommerce_review_order_before_payment_methods` action in `payments.php` template.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Added 'woocommerce_review_order_before_payment_methods' action in 'payments.php' template.

### CODE REVIEW

File: templates/checkout/payment.php [hunk 1]
```diff
@@ -22,6 +22,9 @@
 }
 ?>
 <div id="payment" class="woocommerce-checkout-payment">
+
+	<?php do_action( 'woocommerce_review_order_before_payment_methods' ); ?>
```
Comment (jorgeatorres) [inline templates/checkout/payment.php:26]: Can we please add a docblock for this new action?

**Review by jorgeatorres — CHANGES_REQUESTED** [review]
Hi @manospsyx!

Thanks for the PR 😺.

Overall this looks good, but it needs to be rebased to fix conflicts before we can merge and a docblock needs to be added for the new action.
Also, for context, it'd be great if you described the use case here and possibly add some testing instructions or a snippet to the PR.

Thanks again!

**Comment by jorgeatorres** [comment]
Closing due to inactivity. @manospsyx: please let me know if your team still needs this change and I'll happily take over.

**Comment by manospsyx** [comment]
@xristos3490 do you still need this one?

**Pull request closed without merging**